### PR TITLE
Created utility class for simple conversions

### PR DESCRIPTION
**Pull request by kgonzago**
@csmoore Please review when you get a chance. This PR is part of the larger task #470 

(Updated CSM) - test with - https://github.com/Esri/visibility-addin-dotnet/pull/263

### CODE REVIEW

File: source/CoordinateConversion/CoordinateConversionLibrary/Helpers/ConversionUtils.cs [hunk 1]
```diff
@@ -0,0 +1,71 @@
+﻿using System;
```
Comment (csmoore) [inline source/CoordinateConversion/CoordinateConversionLibrary/Helpers/ConversionUtils.cs]: Just needs a file header

**Comment by csmoore** [comment]
@kgonzago - looks good - you may want to wait to merge until we can test/validate with the other repos? Your call - PR is approved.

**Comment by kgonzago** [comment]
@csmoore I'll add the header info. You're right. let's test first with Viz and D&D before merging.

**Comment by kgonzago** [comment]
@csmoore Tested and verified with Viz add-in behaves as expected when referencing the CC library conversion function. Can you review the two commits I added after your first review? Thanks.

**Pull request merged**